app/views/notifications/_article.html.erb

```diff
@@ -1,59 +1,58 @@
-<% if notification.action == "Reaction" %>
-  <%= render "reaction", notification: notification %>
-<% elsif notification.action.include? "Milestone" %>
-  <%= render "milestone", notification: notification %>
-<% else %>
-  <% json_data = notification.json_data %>
-  <% if json_data["organization"] %>
-    <div class="article-organization-headline">
-      <a class="org-headline-filler" href="<%= json_data["organization"]["path"] %>">
-        <img alt="<%= json_data["organization"]["name"] %> logo" src="<%= json_data["organization"]["profile_image_90"] %>">
-        <%= json_data["organization"]["name"] %>
-      </a>
-   </div>
-  <% end %>
-  <a href="<%= json_data["user"]["path"] %>" class="small-pic-link-wrapper">
-    <div class="small-pic">
-      <img src="<%= json_data["user"]["profile_image_90"] %>" alt="link to <%= json_data["user"]["username"] %>'s profile">
-    </div>
-  </a>
-  <div class="content notification-content article-content">
-    <a href="<%= json_data["user"]["path"] %>">
-      <%= json_data["user"]["name"] %>
-    </a>
-    made a new post<%= ":" unless json_data["organization"] %>
-    <% if json_data["organization"] %>
-      under <a href="<%= json_data["organization"]["path"] %>"><%= json_data["organization"]["name"] %></a>:
-    <% end %>
-
-    <div>
-      <small><%= time_ago_in_words json_data["article"]["published_at"] %> ago</small>
-    </div>
+<div class="crayons-card notification">
+  <% if notification.action == "Reaction" %>
+    <%= render "reaction", notification: notification %>
+  <% elsif notification.action.include? "Milestone" %>
+    <%= render "milestone", notification: notification %>
+  <% else %>
+    <% json_data = notification.json_data %>
+    <%= render "notifications/shared/profile_pic", json_data: json_data %>
 
-    <a href="<%= json_data["article"]["path"] %>">
-      <div class="notification-new-post">
-        <div class="notification-new-post-title">
-          <%= h(json_data["article"]["title"]) %>
-        </div>
-        <div class="notification-new-post-tags">
-          <% json_data["article"]["cached_tag_list_array"].each do |tag| %>
-            #<%= tag %>
+    <div class="notification__content">
+      <header class="mb-4">
+        <h2 class="fs-base fw-normal">
+          <a href="<%= json_data["user"]["path"] %>" class="crayons-link fw-bold">
+            <%= json_data["user"]["name"] %>
+          </a>
+          made a new post<%= ":" unless json_data["organization"] %>
```

Comment:
I am thinking that the ":" that we add at the end of each sentence is not necessary, and would advocate to get rid of it. What do you think?

Reply:
No strong opinion so dropping the `:`.